doc/start/os-recommendations.rst

```diff
@@ -49,7 +49,7 @@ non-Linux systems but these are not supported by the core Ceph effort.
 +===============+===============+=================+==================+==================+
 | Centos 7      |               |                 |         A        |      B           |
 +---------------+---------------+-----------------+------------------+------------------+
-| Centos 8      |    A          |     A           |         A        |      A           |
+| Centos 8      |               |                 |                  |                  |
```

Comment:
Should Centos 9 line receive an A for Quincy?

Reply:
"C" probably? we've been building centos 9 packages for quincy but it's not part of our qa: https://github.com/ceph/ceph/tree/quincy/qa/distros/supported

Reply:
Agreed with @cbodley

Reply:
FYI here is the PR adding c9 support to the quincy suite for those interested in reviewing: https://github.com/ceph/ceph/pull/58104

Reply:
Note from today's CLT meeting:
> - https://github.com/ceph/ceph/pull/58092 (Zac)
>   - as no more discussion on moving away from centos has taken place,
>    and for now we are effectively replacing centos 8 with centos 9 in
>    the suites, centos 9 should be listed as tested (status A, starting
>    from 17.2.8 for quincy and 18.2.3 for reef)

A lot of people have approved the PR as is, but IMO this should be incorporated here to avoid it slipping through the cracks in the coming weeks.

Reply:
@neha-ojha

Reply:
+1